### DISCUSSION TITLE
Does the licence permit outline to a part of prebuilt machine?

### DISCUSSION THREAD

**Discussion by PovilasID** [outline/outline#4375] [General] [answered]
Hey,

I have a project in which I a take a number of software products package them on specific hardware to work well together and ship a device to a client. 
I want to include Outline as documentation for other software on it but... since it would be out of my hands the client would get the device and would and full control of instance of outline they would get "access the functionality of the Licensed Work by creating teams and documents controlled by such third parties."
I am no lawyer but it seams to me that if you read it 'as written' it conflicts with the licence but 'as intended' maybe not?+
Can you clear this up for me?

**Comment by tommoor** [comment, marked as answer]
What is the project? Do you charge for any aspect? 

Through your usage of “client” language it _sounds_ like this would be a breach of the license.

**Reply by PovilasID** [reply]
It's hardware and software that is preconfigured to host information systems for specific purposes of a client. 
Customers are likely to make alterations to documentation as the times goes one but are very unlikely to share it online or have them  accessible to the public internet. 
They pay for the device and the software configuration service. 

Reason I may have though that it may not be in breach is because it does not create competition for your hosted services. My customers are legally limited in use of 3rd party services. In short, they have not bought a hosted service in a last decade.

I was just hoping to avoid using different systems for personal and professional use but thanks for clarification.
